modules/nw-sriov-dual-nic-con.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: CONCEPT
+[id="nw-sriov-dual-nic-con_{context}"]
+= NIC partitioning for SR-IOV devices (Technology Preview)
+
+{product-title} can be deployed on a server with a dual port network interface card (NIC).
+You can partition a single, high-speed dual port NIC into multiple virtual functions (VFs) and enable SR-IOV.
+The following diagram shows how this works:
```

Comment:
There is no diagram here. I know it was already noted in another comment, but that one was resolved and the diagram is still missing.